students/ptfbanks/Lesson01/breakme.py

```diff
@@ -0,0 +1,20 @@
+#syntaxerror
+print string
+
+#Name error
+y=a
+x=45
+print (y+X)
+
+# indentation Errror
+ int (a)
+b= "try this"
+print (a+b)
+
+#type error
+b=3
+txt= b[:2]
+
+#Attribute error
+x=3
+x.append (2)
```

Comment:
recommend installing a linter for your editor or running it separate; this will catch pep8 style guideline violations for python.

some examples:
`x.append (2)` -> `x.append(2)`
`b=3` -> `b = 3`